src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,164 @@
+package duke;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a parser to parse inputs into actions to be taken.
+ */
+public class Parser {
+    private String bye = "bye";
```

Comment:
I like how you define the command strings as a class field. You can also consider making these constant, since the commands are not likely to change over the course of running the program (unless you support command remapping).

Reply:
Yes, it should have been private static final instead, I will change them.
Thank you!